tests/Unit/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.py

```diff
@@ -131,6 +131,12 @@ def spacetime_deriv_detg(sqrt_det_spatial_metric, inverse_spatial_metric,
     return dg
 
 
+def extrinsic_curvature(spacetime_normal_vector, pi, phi):
+    return 0.5 * pi[1:, 1:] + 0.5 * np.einsum(
```

Comment:
Suggest parens to make the formatting less odd.

Reply:
added!